pulp_rpm/app/models.py

```diff
@@ -736,20 +746,47 @@ class PackageGroup(Content):
     display_order = models.IntegerField()
     name = models.CharField(max_length=255)
     description = models.TextField()
-    packages = models.TextField()
+    packages = models.TextField(default='[]')
 
     biarch_only = models.BooleanField(default=False)
 
-    desc_by_lang = models.TextField()
-    name_by_lang = models.TextField()
+    desc_by_lang = models.TextField(default='[]')
```

Comment:
Are these fields a JSON list? If so, I will include these changes on my PR(https://github.com/pulp/pulp_rpm/pull/1458), then my PR can only be merged after this one.

Reply:
I don't think any of my fields are JSON fields.  I will check, but you should go ahead and merge your PR and I can always rebase and adjust here if necessary.

Reply:
it has a data migration, so it is better if I rebase with yours

Reply:
A question about data migration https://github.com/pulp/pulp_rpm/pull/1458/files#r329913404

Reply:
agreed, I will change it there, thanks @goosemania